cmd/generate/dockerfile-templates/BuildImageDockerfile.template

```diff
@@ -1,16 +1,17 @@
 # DO NOT EDIT! Generated Dockerfile.
 
+FROM registry.ci.openshift.org/ocp/4.17:cli-artifacts as tools
+
 # Dockerfile to bootstrap build and test in openshift-ci
 FROM {{.builder}} as builder
 
-RUN echo "[kubernetes]" >> /etc/yum.repos.d/kubernetes.repo && \
-    echo "name=Kubernetes" >> /etc/yum.repos.d/kubernetes.repo && \
-    echo "baseurl=https://pkgs.k8s.io/core:/stable:/v1.29/rpm/" >> /etc/yum.repos.d/kubernetes.repo && \
-    echo "enabled=1" >> /etc/yum.repos.d/kubernetes.repo && \
-    echo "gpgcheck=1" >> /etc/yum.repos.d/kubernetes.repo && \
-    echo "gpgkey=https://pkgs.k8s.io/core:/stable:/v1.29/rpm/repodata/repomd.xml.key" >> /etc/yum.repos.d/kubernetes.repo
+ARG TARGETARCH
+
+COPY --from=tools /usr/share/openshift/linux_$TARGETARCH/oc.rhel8 /bin/oc
```

Comment:
`/bin/oc => /usr/bin/oc` since following lines assume that path

Reply:
Good catch. Updated